fvtest/compilertriltest/OpCodeTest.hpp

```diff
@@ -46,20 +46,28 @@ namespace TRTest
  * returns the expected return value of the opcode test when given the input
  * values from the first part of the outer tuple.
  */
-template <typename Ret, typename... Args>
-using ParamType = std::tuple<std::tuple<Args...>, std::tuple<std::string, Ret (*)(Args...)> >;
+
+// TODO: This is a great idea but unfortunately requires C++11 support.
+//       Emulating the same behaviour without the required C++11 features is 
+//       nontrivial and only has the opposite effect in complicating the code
+//       further. Hence this has been commented out. We should eventually turn
+//       this back on, once we have moved to a C++11 supported compilers (while
+//       also sifting through the Test Instances and using lambdas where appropriate).
+//template <typename Ret, typename... Args>
+//using ParamType = std::tuple<std::tuple<Args...>, std::tuple<std::string, Ret (*)(Args...)> >;
 
 /**
  * @brief Type for holding argument to parameterized binary opcode tests
  */
-template <typename Ret, typename Left, typename Right>
-using BinaryOpParamType = ParamType<Ret, Left, Right>;
 
-/**
- * @brief Struct equivalent to the BinaryOpParamType tuple
- *
- * Used for easier unpacking of test argument.
- */
+//template <typename Ret, typename Left, typename Right>
+//using BinaryOpParamType = ParamType<Ret, Left, Right>;
+
+
+
+// TODO: Once the above has been done we should modify the function signatures
+//       appropriately.
```

Comment:
I presume the unstated action here is an upgrade of the build compiler?

If so, I renew my objection to using TODOs in the code, and in particular for some event (enabling code) that must happen when another event (upgrading the build compiler) occurs. Please create an issue for this so that this action and rationale won't be lost in the code, and reference the issue here instead.  This could very well be the same issue for all the build compiler TODOs in this PR.

If not, please explain.